src/Algebra/Graph/Acyclic/AdjacencyMap.hs

```diff
@@ -0,0 +1,58 @@
+module Algebra.Graph.Acyclic.AdjacencyMap where
```

Comment:
We need to add an export list -- at least the constructor will have to be hidden.

Reply:
Yup! I wanted to do all of that in the end. Will do it in the next commit.